src/templates/video.tsx

```diff
@@ -0,0 +1,203 @@
+    width: 100%;
+    @media ${screenSize.largeAndUp} {
+        margin: initial;
+    }
+`;
+
+const Icons = styled('div')`
+    margin: 0 ${size.small};
+    span {
+        &:not(:first-of-type) {
+            margin-left: ${size.small};
+        }
+        padding: 0 ${size.tiny};
+    }
+    @media ${screenSize.largeAndUp} {
+        display: flex;
+        flex-direction: column;
+        margin: 0 ${size.default};
+        span:not(:first-of-type) {
+            margin-top: ${size.small};
+        }
+    }
+`;
+
+const StyledParagraph = styled(P)`
+    line-height: ${lineHeight.default};
+`;
+
+const dateFormatOptions = {
+    month: 'short',
+    day: '2-digit',
+    year: 'numeric',
+    timeZone: 'UTC',
+};
+
+type VideoProps = {
+    pageContext: {
+        data: IVideo;
+    };
+};
+
+const Video = ({
+    pageContext: {
+        data: {
+            slug,
+            description,
+            publishDate,
+            thumbnailUrl,
+            title,
+            videoId,
+            mediaType,
+        },
+    },
+}: VideoProps) => {
+    const { siteUrl } = useSiteMetadata();
+    const pageUrl = addTrailingSlashIfMissing(`${siteUrl}${slug}`);
+    const formattedPublishDate = toDateString(publishDate, dateFormatOptions);
+    const capitalizedBreadcrumb = mediaType.charAt(0).toUpperCase() + mediaType.slice(1);
+
+    const videoBreadcrumb = useMemo(
+        () => {
+            const videoBreadCrumb = [
+            ...VIDEO_BREADCRUMB,
+                {
+                    label: `${capitalizedBreadcrumb} Video`,
+                    target: `/type/${mediaType}`,
+                },
+                {
+                    label: title,
+                    target: slug
+                }
+                ];
+            return videoBreadCrumb;
+        },
+        [mediaType, slug]
+    );
+
+    const image = useMemo(
+        () =>
+            mediaType === 'twitch'
+                ? getTwitchThumbnail(thumbnailUrl)
+                : thumbnailUrl,
+        [mediaType, thumbnailUrl]
+    );
+
+    return (
+        <Layout includeCanonical={false}>
+            <SEO
+                articleTitle={title}
+                canonicalUrl={pageUrl}
+                image={image}
+                metaDescription={description}
+                ogDescription={description}
+                ogTitle={title}
+                ogUrl={pageUrl}
+                twitter={{
+                    description,
+                    image,
+                    title,
+                }}
+            />
+            <ArticleSchema
+                articleUrl={pageUrl}
+                title={title}
+                description={description}
+                publishedDate={formattedPublishDate}
+                imageUrl={image}
+            />
+            <BlogPostTitleArea
+                breadcrumb={videoBreadcrumb}
+                originalDate={formattedPublishDate}
+                title={title}
+            />
```

Comment:
Do we also want to put the thumbnail here?

Reply:
These are important for search engines and social sharing actually. They add some metadata tags which help these pages to be properly indexed as well as show a nice preview when sharing to linkedin, facebook, etc. We should definitely keep them and they looked good to me.

Reply:
These components add fields such as https://ogp.me/